### ISSUE TITLE
Drag and drop image result in repeated images inserted

### ISSUE THREAD

**Issue by oeddyo** [strapi/strapi#1165]
**Informations**
- **Node.js version**: 9.0.0
- **npm version**: 5.5.1
- **Strapi version**: 12.2.2
- **Database**: mongo
- **Operating system**: mac os

**What is the current behavior?**
When I drag and drop an image in WYSIWYG editor, sometimes, it shows "Uploading" as one line, and when it's done uploading, the "uploading" line is still there.
![image](https://user-images.githubusercontent.com/1943146/39915810-78011c04-553b-11e8-93a2-007b007593bc.png)

**Steps to reproduce the problem**
Just drag and drop an image. But it's not consistently reproduced every time. So try a couple of times. Maybe try to limit chrome's upload speed

**What is the expected behavior?**
only the uploaded url image should be listed 

**Suggested solutions**

---

- [Y ] I'm sure that this feature hasn't already been referenced.

**Comment by oeddyo** [comment]
hmm... the user experience here could use some improvements since when you're trying to upload multiple images, you want to bang your head against the desk  : D

![image](https://user-images.githubusercontent.com/1943146/39916226-799be14c-553c-11e8-82fe-938ac17db85b.png)

**Comment by lauriejim** [comment]
Thank you for this report. Probably will not be fixed soon cause it's not critical stuff.

**Comment by oeddyo** [comment]
understood

I tried to report everything we encountered to make the community stronger. 

However I think it is critical if you use strapi for real to manage a realistic project, and I think that should be what Strapi is going right? Not just a cool project but a truly viable project used by real companies and real people :D 

But per your comment, I do agree you guys have more important stuff on your plate

Thanks for your great work!
